frontend/compositions/header/header.module.css

```diff
@@ -0,0 +1,85 @@
+.wrapper {
+  width: 100%;
+  height: 124px;
+  display: grid;
+  position: relative;
+}
+
+.banner {
+  height: 76px;
+  background-image: url(./assets/Banner.svg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  display: grid;
+  grid-template-columns: 167px 549px 225px 100px;
+  grid-template-rows: 124px;
+}
+
+.logo {
+  padding-left: 72px;
+  margin-top: 27px;
+}
+
+.titleContainer {
+  padding-top: 46px;
+  margin-left: 16px;
+}
+
+.titleContainer .title {
+  color: var(--darkBlue);
+  font-weight: 700;
+  font-size: 28px;
+  line-height: 33.89px;
+}
+
+.titleContainer .subtitle {
+  color: var(--black);
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 17px;
+  letter-spacing: -0.3px;
+}
+
+.nav {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  height: 25px;
+  margin-top: 51px;
+}
+
+.nav li {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  color: var(--darkBlue);
+  text-align: center;
+  line-height: 12px;
+  padding: 8px 15px 5px 14px;
+  cursor: pointer;
+}
+
+.tab {
+  background-color: #ffffff;
+  width: 75px;
+  height: 25px;
+  border-radius: 3px 3px 0px 0px;
+  box-shadow: 0px -2px 4px rgba(48, 52, 99, 0.25);
+}
+
+.donate {
+  margin-top: 21px;
+}
+
+.donate button {
```

Comment:
I'm curious why we wrapped this in the `.donate` div instead of applying styles directly to the button!

Reply:
Thank you for the catch  -  will apply styles to button directly.